backend/canisters/user_index/api/src/updates/assign_platform_moderators_group.rs

```diff
@@ -0,0 +1,29 @@
+use candid::{CandidType, Principal};
```

Comment:
I thought we only needed to call this method locally

Reply:
Done